### PR TITLE
cli: checking config file exist or not.

### PR DESCRIPTION
**Pull request by Txiaozhe**
Could I add unit test for cli in /packages/cli/test, and use 'mocha' as tool.

### CODE REVIEW

**Comment by yorkie** [comment]
> Could I add unit test for cli in /packages/cli/test, and use 'mocha' as tool.

Sorry, we use jasmine as the testing framework.

File: packages/cli/lib/start.js [hunk 1]
```diff
@@ -9,6 +10,11 @@ const start = async (filename) => {
     return;
   }
 
+  if (!fs.existsSync(filename)) {
+    spinner.fail(`Config file: '${filename}' not exist.`);
```
Comment (yorkie) [inline packages/cli/lib/start.js:14]:
```suggestion
    spinner.fail(`${filename} not exists`);
```

**Comment by Txiaozhe** [comment]
> > Could I add unit test for cli in /packages/cli/test, and use 'mocha' as tool.
> 
> Sorry, we use jasmine as the testing framework.

So could I add unit test for cli in /packages/cli/test?

**Comment by yorkie** [comment]
> So could I add unit test for cli in /packages/cli/test?

I'm not sure if this is the right time, because cli is still not using the TypeScript, maybe the cli unit tests can be added after that.

**Comment by Txiaozhe** [comment]
> > So could I add unit test for cli in /packages/cli/test?
> 
> I'm not sure if this is the right time, because cli is still not using the TypeScript, maybe the cli unit tests can be added after that.

OO, it is reasonable :)

**Comment by yorkie** [comment]
And also that would be great if you help us to rewrite cli in TypeScript :)

**Comment by Txiaozhe** [comment]
> And also that would be great if you help us to rewrite cli in TypeScript :)

I’d love to and could you please create an issue and assign it to me? :)

**Pull request merged**